packages/react/src/components/Forms/Select/index.tsx

```diff
@@ -0,0 +1,342 @@
+                        downshiftReturn.selectItem(null);
+                      }}
+                      icon={<span className="material-icons">close</span>}
+                    />
+                    <div className="c__select__inner__actions__separator" />
+                  </>
+                )}
+
+                <Button
+                  color="tertiary"
+                  size="small"
+                  className="c__select__inner__actions__open"
+                  icon={
+                    <span
+                      className={classNames("material-icons", {
+                        opened: downshiftReturn.isOpen,
+                      })}
+                    >
+                      arrow_drop_down
+                    </span>
+                  }
+                  disabled={disabled}
+                  {...downshiftReturn.toggleButtonProps}
+                />
+              </div>
+            </div>
+          </LabelledBox>
+        </div>
+        <div
+          className={classNames("c__select__menu", {
+            "c__select__menu--opened": downshiftReturn.isOpen,
+          })}
+          {...downshiftReturn.getMenuProps()}
+        >
+          <ul>
+            {downshiftReturn.isOpen &&
+              options.map((item, index) => (
+                <li
+                  className={classNames("c__select__menu__item", {
+                    "c__select__menu__item--highlight":
+                      downshiftReturn.highlightedIndex === index,
+                    "c__select__menu__item--selected":
+                      downshiftReturn.selectedItem === item,
+                  })}
+                  key={`${item.value}${index}`}
+                  {...downshiftReturn.getItemProps({ item, index })}
+                >
+                  <span>{item.label}</span>
+                </li>
+              ))}
+          </ul>
+        </div>
+      </div>
+    </Field>
+  );
+};
+
+const SelectSimple = (props: SubProps) => {
+  const downshiftReturn = useSelect({
+    ...props.downshiftProps,
+    items: props.options,
+    itemToString,
+  });
+  const [labelAsPlaceholder, setLabelAsPlaceholder] = useState(
+    !downshiftReturn.selectedItem
+  );
+
+  useEffect(() => {
+    setLabelAsPlaceholder(!downshiftReturn.selectedItem);
+  }, [downshiftReturn.selectedItem]);
+
+  return (
+    <SelectAux
+      {...props}
+      downshiftReturn={{
+        ...downshiftReturn,
+        wrapperProps: downshiftReturn.getToggleButtonProps({
+          disabled: props.disabled,
+        }),
+        toggleButtonProps: {},
+      }}
+      labelAsPlaceholder={labelAsPlaceholder}
+    >
+      {downshiftReturn.selectedItem && (
+        <span>{itemToString(downshiftReturn.selectedItem)}</span>
+      )}
+    </SelectAux>
+  );
+};
+
+const SelectSearchable = (props: SubProps) => {
+  const [optionsToDisplay, setOptionsToDisplay] = useState(props.options);
+  const [inputFocus, setInputFocus] = useState(false);
```

Comment:
By convention, we start boolean state name by a verb.
```suggestion
  const [hasInputFocused, setHasInputFocused] = useState(false);
```